internal/util/rt/v1/add.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	goerrors "errors"
+
+	"github.com/ceph/ceph-csi/internal/util/rt/errors"
+	"github.com/ceph/ceph-csi/internal/util/rt/radoswrapper"
+	"github.com/ceph/ceph-csi/internal/util/rt/reftype"
+)
+
+// Atomically adds refs to an existing RT object.
```

Comment:
instead of operation-specific file names can we have everything in one file with interfaces and methods which will help us to mock in when writing the Unit test cases.